### PR TITLE
🥅 Guard decay info and airdrop end message

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

File: locales/en.json [hunk 1]
```diff
@@ -62,7 +63,7 @@
   "AirDrop.mission.button.previous": "Previous",
   "AirDrop.mission.button.retry": "retry",
   "AirDrop.mission.button.seeTransaction": "Check Tx",
-  "AirDrop.mission.button.technicalError": "Please try again in a few minutes.",
+  "AirDrop.mission.button.technicalError": "Not opened for claim",
```
Comment (nwingt) [inline locales/en.json:66]:
```suggestion
  "AirDrop.mission.button.technicalError": "Not open to claim",
```

File: pages/airdrop/claim.vue [hunk 2]
```diff
@@ -303,7 +315,7 @@ export default class AirdropClaimPage extends Vue {
     this.missionLoadingStatus = 'Loading'
     const res: any = await this.$axios
       .post(
```
Comment (williamchong) [inline pages/airdrop/claim.vue:317]: Can we block this action if airdrop end is true?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: pages/airdrop/claim.vue (hunk 2 above)
Comment (WeiJun0827) [reply pages/airdrop/claim.vue:317]: Blocked, also blocked from the server

File: pages/airdrop/claim.vue [hunk 3]
```diff
@@ -161,8 +159,12 @@ export default class AirdropClaimPage extends Vue {
     return `${BIG_DIPPER_TX_BASE_URL}${this.currentMission.txHash}`
   }
 
-  mounted() {
-    this.fetchMissionDecay()
+  get isAirdropEnd() {
+    return this.decay.factor === 1;
```
Comment (williamchong) [inline pages/airdrop/claim.vue:163]:
```suggestion
    return this.decay.factor >= 1;
```
In case the value somehow > 1

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by williamchong — APPROVED** [review]

**Pull request merged**